### PR TITLE
Use the number of seconds the tasks were running to get the number of allowed traces

### LINKED ISSUE #610: RateLimiter stress test is failing occasionally
The RateLimiter stress test is failing sometimes:
https://ci.appveyor.com/project/GoogleCloudPlatform/google-cloud-dotnet/build/1.0.463-appveyor

The failure seems to only be one extra trace.

I can't seem to repo the issue with the current test on linux or windows.

### PR DESCRIPTION
**Pull request by iantalarico**
Using a predetermined count was causing intermittent failures if the tasks ran longer than expected.

Fixes #610

### CODE REVIEW

File: apis/Google.Devtools.AspNet/Google.Devtools.AspNet.Tests/CloudTrace/RateLimiterTest.cs [hunk 1]
```diff
@@ -69,6 +71,10 @@ public void CanTrace_Multiple()
                 int canTraceCounter = 0;
                 int threads = 10;
 
+                // Start a timer to get the total time from tasks starting to ending.
+                Stopwatch timer = new Stopwatch();
+                timer.Start();
```
Comment (evildour) [inline apis/Google.Devtools.AspNet/Google.Devtools.AspNet.Tests/CloudTrace/RateLimiterTest.cs]: This is a bit cleaner: `var timer = Stopwatch.StartNew();` (or with the type name if you prefer).

File: apis/Google.Devtools.AspNet/Google.Devtools.AspNet.Tests/CloudTrace/RateLimiterTest.cs [hunk 2]
```diff
@@ -86,9 +92,16 @@ public void CanTrace_Multiple()
                     });
                 }
 
-                // Set a timeout of 2.1 seconds which should allow 2 traces.
+                // Set a timeout to 2.1 seconds which should allow 2 traces unless control
+                // of the tasks does not return right away.  To ensure a proper trace count
+                // use the number of seconds to check the trace count.
                 await Task.Delay(2100);
-                Assert.Equal(2, canTraceCounter);
+                var elapsedSeconds = Math.Floor(timer.ElapsedMilliseconds / 1000.0);
```
Comment (evildour) [inline apis/Google.Devtools.AspNet/Google.Devtools.AspNet.Tests/CloudTrace/RateLimiterTest.cs]: Use `Math.Floor(timer.Elapsed.TotalSeconds)` instead.

Comment (iantalarico) [inline apis/Google.Devtools.AspNet/Google.Devtools.AspNet.Tests/CloudTrace/RateLimiterTest.cs]: Done

**Comment by iantalarico** [comment]
Thanks Mike, PTAL

**Pull request merged**